src/t_hash.c

```diff
@@ -589,7 +609,8 @@ SetExRes hashTypeSetExpiryListpack(HashTypeSetEx *ex, sds field,
     }
 
     if (prevExpire == EB_EXPIRE_TIME_INVALID) {
-        if (ex->expireSetCond & (HFE_XX | HFE_LT | HFE_GT))
+        /* For fields without expiry, LT condition is considered valid */
```

Comment:
you may add a test for this case

Reply:
Already have one. Please see `test "HPEXPIRE(AT) - Test 'LT' flag ($type)"`